ui/app/helpers/utils/metametrics.util.js

```diff
@@ -62,8 +62,8 @@ const customDimensionsNameIdMap = {
   [METAMETRICS_CUSTOM_ENVIRONMENT_TYPE]: 6,
   [METAMETRICS_CUSTOM_ACTIVE_CURRENCY]: 7,
   [METAMETRICS_CUSTOM_ACCOUNT_TYPE]: 8,
-  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 9,
-  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 10,
+  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 10,
+  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 9,
```

Comment:
Could we sort these?

```suggestion
  [METAMETRICS_CUSTOM_NUMBER_OF_ACCOUNTS]: 9,
  [METAMETRICS_CUSTOM_NUMBER_OF_TOKENS]: 10,
```

Reply:
Done, and I updated the order of the constants to reflect this order.